<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RemoveRedundantConfirmationForm_template</class>
 <widget class="QDialog" name="RemoveRedundantConfirmationForm_template">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>693</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Instructions and confirmation of remove redundant constraints</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="4">
    <widget class="QPlainTextEdit" name="plainTextEdit"/>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="dontShowAgainCheckBox">
     <property name="text">
      <string>Don't show this again</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>607</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="continuePushButton">
     <property name="text">
      <string>Continue</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="cancelPushButton">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
